brunch-config.js

```diff
@@ -160,9 +160,19 @@ exports.config = {
     assetsmanager: {
         copyTo: {
           '/templates/' : [ 'src/app/templates/*.tmpl' ],
+          '/cms/api/' : [ 'src/cms/api/*.php' ],
```

Comment:
brunch will ignore php files, unless they are in assets folder .. so we manually copy them to where we want them